spec/models/move_spec.rb

```diff
@@ -558,20 +558,18 @@
     end
   end
 
-  describe '.updated_at_range' do
-    let(:updated_at_from) { Time.zone.now.beginning_of_day - 1.day }
-    let(:updated_at_to) { Time.zone.now.end_of_day - 1.day }
-
-    let!(:before_start_move) { create(:move, updated_at: updated_at_from - 1.second) }
-    let!(:on_start_move) { create(:move, updated_at: updated_at_from) }
-    let!(:on_end_move) {  create(:move, updated_at: updated_at_to) }
-    let!(:after_end_move) { create(:move, updated_at: updated_at_to + 1.second) }
+  describe '.updated_at_range scope' do
+    let(:updated_at_from) { Time.zone.yesterday.beginning_of_day }
+    let(:updated_at_to) { Time.zone.yesterday.end_of_day }
 
     it 'returns the expected moves' do
-      actual_moves = described_class.updated_at_range(
-        updated_at_from,
-        updated_at_to,
-      )
+      create(:move, updated_at: updated_at_from - 1.second)
```

Comment:
Can you make this same refactor to people/profiles? I agree it's cleaner/clearer :)

Reply:
sure, even though, we are using the same scope there, I'd suggest  (not now) to clean this up, we got 4 identical scopes in 4 different models, and we test it in all the places, it sounds we should use a generic scope, include it in the models, and test just once

Reply:
I was thinking the same thing earlier today :)